### DISCUSSION TITLE
Suggestions for the net speed indicator feature

### DISCUSSION THREAD

**Discussion by JediRhymeTrix** [tribalfs/GalaxyMaxHzPub#221] [Ideas]
- Quick settings toggle to show/hide indicator
- Option to hide the status bar indicator without having to minimize the notification

**Comment by tribalfs** [comment]
First one is already available since 8.1.7*.
I am not sure what you meant on the 2nd one.

**Reply by JediRhymeTrix** [reply]
I'm on 8.1.25 and don't see a quick settings toggle for the network speed indicator.

In the second suggestion, I just mean that right now, there's an icon in the status bar that shows the network speed. I would like there to be an option to hide this icon and only have the bar in the notification shade.

**Reply by tribalfs** [reply]
Install the latest version. https://github.com/tribalfs/GalaxyMaxHzPub/releases

I don't think 2nd is possible.

**Comment by JediRhymeTrix** [comment]
Fair enough. Thanks.
